### PR TITLE
Ledger DDL.

### PR DESCRIPTION
**Pull request by thjhc**

### CODE REVIEW

File: ledger.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CREATE  TABLE ledger
+(
+        PRIMARY KEY (ledger_id),
+
+        ledger_id CHAR(10) NOT NULL,
+			CONSTRAINT ledger_id_not_blank
+			CHECK (ledger_id <> '')
```
Comment (ben-mulhern) [inline ledger.sql]: Comma required after constraint, and not required between ledger_id and constraint.

File: ledger.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+        description VARCHAR(50) NULL
```
Comment (ben-mulhern) [inline ledger.sql]: I'd recommend making this NOT NULL and having a constraint that doesn't let it be blanks.

File: ledger.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+CREATE  TABLE ledger
+(
+        PRIMARY KEY (ledger_id),
+
+        ledger_id CHAR(10) NOT NULL,
```
Comment (williamthurley) [inline ledger.sql]: What is the reasoning behind a 10 character id?

File: ledger.sql [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+			CONSTRAINT ledger_id_not_blank
+			CHECK (ledger_id <> '')
+
+        description VARCHAR(50) NULL
+              
+)
```
Comment (ben-mulhern) [inline ledger.sql]: In general I'd add a semi-colon at the end of the statement.

**Review by williamthurley — COMMENTED** [review]
All good, just want to hear the ledger_id reasoning.

**Review by ben-mulhern — APPROVED** [review]
Looks good, will merge.

**Pull request merged**